packages/app/src/cli/services/dev/processes/setup-dev-processes.test.ts

```diff
@@ -21,7 +21,13 @@ import {Config} from '@oclif/core'
 
 vi.mock('../../context/identifiers.js')
 vi.mock('@shopify/cli-kit/node/session.js')
-vi.mock('../fetch.js')
+vi.mock('../fetch.js', async () => {
+  const fetch: any = await vi.importActual('../fetch.js')
+  return {
+    ...fetch,
+    fetchAppExtensionRegistrations: vi.fn(),
+  }
+})
```

Comment:
Why do you need the real `fetch` here? nothing else changed in this file 🤔

Reply:
Good catch! I copied and pasted this from another test :)